torchmultimodal/modules/layers/normalizations.py

```diff
@@ -45,3 +46,25 @@ def forward(self, x: Tensor) -> Tensor:
             self.eps,
         )
         return output.type_as(x)
+
+
+class RMSNorm(nn.Module):
+    """Root Mean Square layer normalization
+    as proposed in: https://arxiv.org/abs/1910.07467
+
```

Comment:
i think this is copy paste of llama. can u add a link

Reply:
added